public/app/features/dashboard-scene/edit-pane/ElementEditPane.tsx

```diff
@@ -23,6 +23,8 @@ export function ElementEditPane({ element }: Props) {
           title={element.typeName}
           isOpenDefault={true}
           className={styles.noBorderTop}
+          renderTitle={element.renderTitle}
+          alwaysExpanded={element.alwaysExpanded}
         >
           <div className={styles.actionsBox}>{element.renderActions()}</div>
         </OptionsPaneCategory>
```

Comment:
I wonder if using OptionsPaneCategory for the element header is the right thing, the element header and actions (And back arrow in the latest designs) feels like a different component